src/instructlab/clickext.py

```diff
@@ -73,25 +73,18 @@ def get_command(self, ctx: click.Context, cmd_name: str) -> click.Command | None
             cmd = self.alias_eps[cmd_name].load()
             if typing.TYPE_CHECKING:
                 assert isinstance(cmd, click.Command)
-            group, primary = self.get_alias_info(cmd_name)
-            click.echo(
-                "You are using an aliased command, this will be deprecated "
-                "in a future release. Please consider using "
-                f"`ilab {group} {primary}` instead"
-            )
             # if some storage dirs do not exist
-            # AND the command is not `ilab init`
             # AND the --config flag is not customized, then we error
             if not storage_dirs_exist() and (
-                primary != "init" and ctx.params["config_file"] == DEFAULTS.CONFIG_FILE
+                ctx.params["config_file"] == DEFAULTS.CONFIG_FILE
             ):
                 click.secho(
                     "Some ilab storage directories do not exist yet. Please run `ilab config init` before continuing.",
                     fg="red",
                 )
                 raise click.exceptions.Exit(1)
             return cmd
-        return super().get_command(ctx, cmd_name)
+        return None
```

Comment:
I don't think you should return None here. The caller expects a command. I think super() works. What is this change trying to resolve?

Reply:
I think None should also be dropped from the typing hint for the return value for this function, seems incorrect. (I understand it's not part of this PR, but maybe you can clean it up.)

Reply:
Resolved this offline-
I reverted it back to `return super().get_command(ctx, cmd_name)`